oc2ls.md

```diff
@@ -749,7 +753,16 @@ The behavior of an implementation receiving an OpenC2 Response with an unsupport
 #### 3.4.1.6 Features
 | Type Name | Base Type | Description |
 | :--- | :--- | :--- |
-| **Features** | ArrayOf(Feature) | An array of zero to ten names used to query an actuator for its supported capabilities. |
+| **Features** | ArrayOf(Feature) [0..10] | An array of zero to ten names used to query an actuator for its supported capabilities. |
+
+**Usage Requirements:**
+
+* A Producer MUST NOT send a list containing more than one instance of any Feature.
+* A Consumer receiving a list containing more than one instance of any Feature MUST behave as if the duplicate(s) were not present.
```

Comment:
I think this should be "A Producer receiving ..."

Reply:
I think it was consumer since the previous sentence says a producer shouldn't ask for more than one. I think this is saying that if a producer does ask for more than one, then the consumer must react anyway like it only sent one. I disagree. I think MUST must change to SHOULD. I disagree with requiring extra code to be added to handle an illegal case a particular way. I think responding with 'illegal command' or just not responding (eg server error) would be a better way to handle.